exporters/otlp/otlptrace/example_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlptrace_test
+
+import (
+	"context"
+	"log"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	instrumentationName    = "github.com/instrumentron"
+	instrumentationVersion = "v0.1.0"
+)
+
+var (
+	tracer = otel.GetTracerProvider().Tracer(
+		instrumentationName,
+		trace.WithInstrumentationVersion(instrumentationVersion),
+		trace.WithSchemaURL("https://opentelemetry.io/schemas/1.2.0"),
+	)
+)
+
+func add(ctx context.Context, x, y int64) int64 {
+	var span trace.Span
+	_, span = tracer.Start(ctx, "Addition")
+	defer span.End()
+
+	return x + y
+}
+
+func multiply(ctx context.Context, x, y int64) int64 {
+	var span trace.Span
+	_, span = tracer.Start(ctx, "Multiplication")
+	defer span.End()
+
+	return x * y
+}
+
+func Resource() *resource.Resource {
+	return resource.NewWithAttributes(
+		semconv.SchemaURL,
+		semconv.ServiceNameKey.String("stdout-example"),
```

Comment:
```suggestion
		semconv.ServiceNameKey.String("otlptrace-example"),
```

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/2053#discussion_r676437996

> I have found out that we have an existing example here: https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector. I was not aware of it - I have found it when I was reviewing your code. I think it would be enough to reference it instead of creating yet another example.

WDYT ?

Reply:
@MrAlias ^